src/main/resources/job-scheduler-config.json

```diff
@@ -76,6 +76,10 @@
       "destinationTableId": "RssUsageByMonth",
       "fileName": "queries/rss-widget-usage-by-month.sql"
     },
+    {
+      "destinationTableId": "TwitterUsageByMonth",
+      "fileName": "queries/twitter-widget-usage-by-month.sql"
```

Comment:
@jungeunyoon do you need to commit those files as well?

Reply:
@jungeunyoon nevermind, I see them now.

Reply:
@rudibravo actually I caught a typo from that comment. thanks. `queries/twitter-widget-usage-by-month.sql` --> `queries/twitter-component-usage-by-month.sql`